spectate/spectate.py

```diff
@@ -82,6 +131,9 @@ def remove_callback(self, name, before=None, after=None):
 
     def wrapper(self, name, args, kwargs):
```

Comment:
Should this be public? If not [this check](https://github.com/rmorshea/spectate/pull/7/files#diff-da697e039a4de4d6f110349ca460c450R147) should be removed.